dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/annocache/AnnotationsBetaHelper.java

```diff
@@ -173,11 +219,17 @@ public static boolean getLibertyBeta() {
         if ( ANNO_BETA_PROPERTY_VALUE != null ) {
             return ANNO_BETA_PROPERTY_VALUE.booleanValue();
         } else {
-            return IS_LIBERTY_BETA_PRODUCT;
+            // Post beta, this test always answers true.
+           return IS_LIBERTY_BETA_PRODUCT;
         }
```

Comment:
If this function is now GA, can this entire method be removed?

Reply:
Yes.  The helper itself should be entirely removed.  The function of the helper should disappear into the annotations adapter code.  However, there are a code locations which access the helper which do not use adapt calls which will need more extensive updates.  These are updated which are intended in the referenced issue.